packages/jellyfish-api-core/__tests__/category/blockchain.test.ts

```diff
@@ -235,6 +235,26 @@ describe('masternode', () => {
     })
   })
 
+  describe('getChainTips', () => {
+    it('should getChainTips', async () => {
+      const chainTips: blockchain.ChainTip[] = await client.blockchain.getChainTips()
+      for (let i = 0; i < chainTips.length; i += 1) {
+        const data = chainTips[i]
+        expect(data.height).toBeGreaterThan(0)
+        expect(typeof data.hash).toBe('string')
+        expect(data.hash.length).toBe(64)
+        expect(data.branchlen).toBeGreaterThanOrEqual(0)
+        expect(
+          data.status === 'invalid' ||
+          data.status === 'headers-only' ||
+          data.status === 'valid-headers' ||
+          data.status === 'valid-fork' ||
+          data.status === 'active'
```

Comment:
use `.includes` to be cleaner

Reply:
Likewise, .includes has the same problem like .toContain

Fuxing: It must always be like this, testing is like reading a narrative.
expect (actual) to be (expected)

Reply:
```ts
expect(['invalid', 'headers-only', ...].includes(data.status)).toBe(true)
```

Reply:
Seems similar to the code I wrote earlier.

`expect(['invalid', 'headers-only', 'valid-headers', 'valid-fork', 'active']).toContain(data.status)`

I followed what was taught here https://jestjs.io/docs/expect#tocontainitem

Reply:
Okay I shall use this -> expect(['invalid', 'headers-only', ...].includes(data.status)).toBe(true). Will commit and push before end of today